kube_metrics_server/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "display_name": "Kube metrics server",
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "kube_metrics_server",
+  "metric_prefix": "kube_metrics_server.",
+  "metric_to_check": "kube_metrics_server.threads",
+  "creates_events": false,
+  "short_description": "Monitors the Kubernetes Metrics Server",
+  "guid": "7a477937-4db8-4277-bd58-9e56ac064185",
+  "support": "core",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "Datadog-Kube_metrics_server Integration",
+  "categories": [
+    "orchestration",
+    "containers"
+  ],
+  "type": "check",
+  "is_public": true,
```

Comment:
by default it should be false I think

Reply:
Yupp, we flip this right before the Agent release that will includes this integration.

Reply:
done, please let me know if you request any other changes :)
@clamoriniere @nmuesch